source/components/Autocomplete.js

```diff
@@ -39,7 +39,7 @@ export default class Autocomplete extends FormField {
     const { selectedOptions, filteredOptions, isOpen } = this.state;
     return Object.assign({}, super.prepareSkinProps(props), {
       selectedOptions,
-      filteredOptions,
+      filteredOptions: filteredOptions || [],
```

Comment:
@DominikGuzei It's worth pointing out that this is fixing a fatal bug, where pressing the backspace key while in an empty `Autocomplete` component would cause a crash.  I'm not sure if you'd rather me break this out to it's own branch or if this is fine.  It is mentioned in the git commit

Reply:
Thanks, i already wanted to ask if you could elaborate on this 👍  great catch! I think it's fine to keep it in this PR for now (since it's just a small change).